proseco/guide.py

```diff
@@ -337,8 +412,15 @@ def check_spoil_contrib(cand_stars, ok, stars, regfrac, bgthresh):
             val = on_region[pixlabel == chandra_aca.aca_image.EIGHT_LABELS][0]
             if val > bgthresh:
                 bg_spoiled[cand_stars['id'] == cand['id']] = True
+                rej.append({'id': cand['id'],
+                            'type': 'region background spoiled',
+                            'bg_thresh': bgthresh,
+                            'bg_pix_val': val,
+                            'pix_id': pixlabel,
+                            'text': f'Cand {cand["id"]} has bg pix spoiled by spoilers'})
+                continue
```

Comment:
This "continue" is another functional change.  If one background pixel is spoiled there's no need to check the rest of them.